frontend/src/lib/components/IntervalFilter/intervalFilterLogic.test.ts

```diff
@@ -0,0 +1,46 @@
+import { initKeaTestLogic } from '~/test/init'
+import { router } from 'kea-router'
+import { expectLogic } from 'kea-test-utils'
+import { intervalFilterLogic } from 'lib/components/IntervalFilter/intervalFilterLogic'
+import { IntervalKeyType } from 'lib/components/IntervalFilter/intervals'
+
+const insightsURL = (
+    additionalPathPart: string = '',
+    dateFrom?: '-90d' | '-14d' | undefined,
+    interval?: IntervalKeyType | undefined
+): string => {
+    let url = `/insights/smosHASp${additionalPathPart}?insight=TRENDS&interval=day&actions=%5B%5D&events=%5B%7B"id"%3A"%24pageview"%2C"name"%3A"%24pageview"%2C"type"%3A"events"%2C"order"%3A0%7D%5D&properties=%5B%5D&filter_test_accounts=false&display=ActionsPie`
```

Comment:
Same with `urls.insightView()`

Reply:
Yep, good call. It's in three files now 💥